tests/conftest.py

```diff
@@ -0,0 +1,178 @@
+"""
+Testing configurations for the `neurostatslib` library.
+
+This module contains test fixtures required to set up and verify the functionality
+of the modules of the `neurostatslib` library.
+
+Dependencies:
+    - jax: Used for efficient numerical computing.
+    - jax.numpy: JAX's version of NumPy, used for matrix operations.
+    - numpy: Standard Python numerical computing library.
+    - pytest: Testing framework.
+    - yaml: For parsing and loading YAML configuration files.
+
+Functions:
+    - poissonGLM_model_instantiation: Sets up a Poisson GLM, instantiating its parameters
+      with random values and returning a set of test data and expected output.
+
+    - poissonGLM_coupled_model_config_simulate: Reads from a YAML configuration file,
+      sets up a Poisson GLM with predefined parameters, and returns the initialized model
+      along with other related parameters.
```

Comment:
not sure these are necessary (and I worry about keeping them up to date) -- why add them?

Reply:
makes sense, i removed most of it